File: Base Dati/esempio.js
let game = {
    nome: "Minecraft Java Edition",
    platform: "pc",
    publisher: "Mojang",
    year: 2011
};

let gameJson = JSON.stringify(game);

console.log(game);
console.log(gameJson);


let game2 = JSON.parse(gameJson);

console.log(game2);

File: Base Dati/esempi/esempio.py
import json

game = {
    "nome": "Minecraft Java Edition",
    "platform": "pc",
    "publisher": "Mojang",
    "year": 2011
}

game_json = json.dumps(game)
game2 = json.loads(game_json)

print(game)
print(game_json)

with open ("game.json", "w", encoding="utf-8") as f:
    json.dump(game_json, f, indent=4)

with open ("game.json", "r", encoding="utf-8") as f:
    oggetto = json.load(f)

print(type(oggetto))
print(oggetto)